Src/AutoFixture/Xeger/Automaton.cs

```diff
@@ -34,7 +34,7 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace AutoFixture.DataAnnotations
+namespace AutoFixture.Xeger
```

Comment:
There's no such thing as `AutoFixture.Xeger`. I think we should either revert this change, or change it to `namespace Fare`, until we're done with #938.